crates/uv/tests/edit.rs

```diff
@@ -74,14 +74,12 @@ fn add_registry() -> Result<()> {
         version = "3.7.0"
         source = "registry+https://pypi.org/simple"
         sdist = { url = "https://files.pythonhosted.org/packages/c6/b3/fefbf7e78ab3b805dec67d698dc18dd505af7a18a8dd08868c9b4fa736b5/anyio-3.7.0.tar.gz", hash = "sha256:275d9973793619a5374e1c89a4f4ad3f4b0a5510a2b5b939444bee8f4c4d37ce", size = 142737 }
+        dependencies = [
+            { name = "idna" },
+            { name = "sniffio" },
```

Comment:
I'm wondering if we should just use a string for them, instead of an object with a single string?

Reply:
This is a good question. Cargo uses strings for dependencies, and I originally wanted to as well, but we have more information that would be straining the limits of "good sense" to cram into a string here. But, in the common case of a dependency just being a single package name? Hmmm.

I think I like this idea. My weak concern is when you have a mixture of inline tables and package name strings. I feel like that might look a little odd and inhibit quick scanning. If you agree that that is odd, then another possibility is to use name strings only when every dependency is a single package name. I think I'd be cool with that.

Reply:
This would be nice, but then introducing the first non-name dependency would result in a massive diff, and removing it again. My main thinking is that there will be few forking cases, so they look like the exception they are between the non-forking cases.